### ISSUE TITLE
Add "nested block comments" check

### ISSUE THREAD

**Issue by NJohnson9402** [tsqllint/tsqllint#310]
In plain TSQL syntax, nested block comments are OK; however, in Liquibase formatted TSQL, they cause a breaking syntax error.

Example:
```
SELECT *
FROM dbo.Foo
/* this is a comment
   /* this is a nested comment
      block of text
   */
   more text
*/
WHERE Name = 'Bar'
```

I understand this repo has nothing to do with Liquibase, but it does highlight the fact that this kind of comment behavior is silly and should be flagged, at least as a warning. Thanks for considering!
